src/scss/dp/overrides/components/_search.scss

```diff
@@ -9,8 +9,16 @@
     height: 80px;
   }
 
+  &--col-border-bottom {
+    border-bottom: 1px solid #000;
```

Comment:
not sure about this app but those usually should be a variables?

Reply:
fixed